amoro-ams/src/main/java/org/apache/amoro/server/authentication/HttpAuthenticationFactory.java

```diff
@@ -40,4 +57,28 @@ private static <T> T createAuthenticationProvider(
       throw new IllegalStateException(className + " must extend of " + expected.getName());
     }
   }
+
+  public static TokenCredential getBearerTokenCredential(
+      Context context, String proxyClientIpHeader) {
+    return new DefaultTokenCredential(
+        getBearerToken(context), getCredentialExtraInfo(context, proxyClientIpHeader));
+  }
+
+  private static String getBearerToken(Context context) {
+    String authorization = context.header(Header.AUTHORIZATION);
+    if (authorization != null) {
```

Comment:
Do we need to add some comments for this

Reply:
done